webserver/views/datasets.py

```diff
@@ -16,9 +16,9 @@
 datasets_bp = Blueprint("datasets", __name__)
 
 
-@datasets_bp.route("/")
-def list_datasets():
-    datasets = db.dataset.get_public_datasets()
+@datasets_bp.route("/<status_value>/<int:page_no>")
```

Comment:
Be careful about this change. Now you can't go to `/datasets` to view the default page. You should fill in defaults if none are provided.